libethereum/ClientTest.cpp

```diff
@@ -60,8 +60,9 @@ void ClientTest::setChainParams(string const& _genesis)
         if (params.sealEngineName != "NoProof")
             BOOST_THROW_EXCEPTION(ChainParamsNotNoProof() << errinfo_comment("Provided configuration is not well formatted."));
 
+        setAuthor(params.author);  // Must be called before reopenChain !!!
```

Comment:
Why must it?

Reply:
see at reopenChain  it replaces the author from m_preSeal. 
so setAuthor would set an author for the m_preSeal

Reply:
yes it should fix author for rpc tests. but we don't run that script anymore.

Reply:
It looks like this `reopenChain` is called only from here.
(also from `Client::killChain()` which looks to be not used and should be removed)

If so, set the author from `params.author` inside `reopwnChain` instead of "backup and restore author" logic there